Update podcasts.md (#4628)
the-turing-way/the-turing-way: book/website/communication/podcasts.md

(cm-podcasts-how)=
## Start a Podcast
There's a podcast series for that! -> [Research Podcasting Essentials](https://open.spotify.com/show/5lmkVt26K5z9IspeCYaiaQ)

You don’t need fancy equipment or expensive software to create a podcast, but there are a few important elements worth planning before you get started.

### Why do you want to start a podast?
Think about the main goal of your podcast. 

> Is it meant to inform, inspire, entertain, raise awareness, or build community around a research topic? Your purpose will guide your content, guests, and overall direction.

Reviewer: ```suggestion
Is it meant to inform, inspire, entertain, raise awareness, or build community around a research topic?
Your purpose will guide your content, guests, and overall direction.
```

You may also want to think about how your podcast connects to your broader initiative or project goals.

_Learn more_<br>
[Episode 1: Why start a research podcast?](https://open.spotify.com/episode/1WpS2sD1YeStqZL9WjZCqR?si=GEBoHYOSTHSEML21uMwKPw)

### What is the structure of your podcast?

> A clear structure helps listeners know what to expect. Consider whether your episodes will follow a consistent format with an intro, discussion, interview, and outro.

Reviewer: ```suggestion
A clear structure helps listeners know what to expect.
Consider whether your episodes will follow a consistent format with an intro, discussion, interview, and outro.
```

You might also want to create templates for show notes and episode scripts to make production easier and create consistency across episodes.

_Learn more_<br>
[Episode 2: Anatomy of a podcast](https://open.spotify.com/episode/12p6bnrcgmvqMEZ7mqzMQ7?si=5iibVBFOTEGngiMsJGfknQ)

### Who is your podcast for?

> Define your target audience early on. Your podcast may be aimed at researchers, students, practitioners, policymakers, or the general public.

Reviewer: ```suggestion
Define your target audience early on.
Your podcast may be aimed at researchers, students, practitioners, policymakers, or the general public.
```

Understanding your audience helps shape the language, tone, level of detail, and topics you choose for your episodes.

_Learn more_<br> 
[Episode 3: Identifying your Audience and Defining your Podcast's Purpose](https://open.spotify.com/episode/4STTYuivPOz8e8itqPu9DG?si=vJdQrWAEQTumlJLchuXiDw)

### Which equipment and technical setup are necessary?
You do not need expensive equipment to start podcasting. 
A simple setup with a laptop or smartphone microphone can work for beginners.

> For higher-quality audio, popular USB microphones include Blue Yeti, Rode, Shure, and Audio-Technica models. 

Reviewer: ```suggestion
For higher-quality audio, popular USB microphones include Blue Yeti, RØDE, Shure, and Audio-Technica models. 
```

For recording interviews, you can use platforms such as Zoom, Teams, Zencastr, or Riverside.fm.

> For editing, beginner-friendly tools include Audacity (free and open source), GarageBand (free on Mac), and Hindenburg Pro or Riverside.fm. Some podcast hosting platforms also include built-in recording and editing tools.

Reviewer: ```suggestion
For editing, beginner-friendly tools include Audacity (free and open source), GarageBand (free on Mac), and Hindenburg Pro or Riverside.fm.
Some podcast hosting platforms also include built-in recording and editing tools.
```

_Learn more_<br>

> [Episode 4: Equipment and Basic Technical Setup on a Budget](https://open.spotify.com/episode/12p6bnrcgmvqMEZ7mqzMQ7?si=5iibVBFOTEGngiMsJGfknQ)

Reviewer: ```suggestion
[Episode 4: Equipment and Basic Technical Setup on a Budget](https://open.spotify.com/episode/1TYCzPc7L2Qx0dSblzVbqo)
```

### What style will your podcast be in?

> Decide on the overall style and format of your show. Your podcast could be interview-based, storytelling-focused, documentary-style, conversational, or educational.

Reviewer: ```suggestion
Decide on the overall style and format of your show.
Your podcast could be interview-based, storytelling-focused, documentary-style, conversational, or educational.
```

You should also think about the tone of the podcast — formal, relaxed, academic, or community-driven — and whether episodes will follow a scripted or more spontaneous format.

_Learn more_<br>
[Episode 5: Interviewing and Storytelling Techniques](https://open.spotify.com/episode/1Xdetu5grUTvNmpCceYLkK?si=Ix5Dj9u2TqqXpn14AzAfmg)

### How to get your podcast out there!
Choosing a hosting platform is an important step in publishing your podcast. 
Popular options include Buzzsprout, Podbean, Spotify for Podcasters, or self-hosting solutions with RSS feeds.

To help people discover your podcast, create engaging show notes, share episodes on social media, invite guests to promote episodes, and include useful resources and links for listeners. 

> You may also want to release your podcast under an open license such as CC-BY 4.0 to encourage sharing and reuse.

Reviewer: ```suggestion
You may also want to release your podcast under an open [license](#rr-licensing), such as CC-BY 4.0, to encourage sharing and reuse.
```
